website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -50,6 +50,28 @@ When materializing a model as `table`, you may include several optional configs
 
 \* Beginning in 1.7.12, we have added tblproperties to Python models via an alter statement that runs after table creation.
 We do not yet have a PySpark API to set tblproperties at table creation, so this feature is primarily to allow users to anotate their python-derived tables with tblproperties.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.8">
```

Comment:
love to see you using these version blocks, but perhaps it's worth calling out at the top of the `1.8` section what's different from the table in the `1.7` section?